lib/WebAssemblyModulesPlugin.js

```diff
@@ -17,8 +17,37 @@ class WebAssemblyModulesPlugin {
 			normalModuleFactory.hooks.createParser.for("webassembly/experimental").tap("WebAssemblyModulesPlugin", () => {
 				return new WebAssemblyParser();
 			});
+
+			compilation.chunkTemplate.hooks.renderManifest.tap("WebAssemblyModulesPlugin", (result, options) => {
+				const chunk = options.chunk;
+				const outputOptions = options.outputOptions;
+				const moduleTemplates = options.moduleTemplates;
+				const dependencyTemplates = options.dependencyTemplates;
+
+				for(const module of chunk.modulesIterable) {
+					if(module.type && module.type.startsWith("webassembly")) {
```

Comment:
I always thought `module.type` couldn't be unset.

Reply:
hmm.. yes, I only moved the code. When this code was written I considered unset module.type for backward-compatibility.